test_runner/regress/test_broken_timeline.py

```diff
@@ -144,7 +143,11 @@ def test_timeline_init_break_before_checkpoint(neon_env_builder: NeonEnvBuilder)
     ), "pageserver should clean its temp timeline files on timeline creation failure"
 
 
-def test_timeline_init_break_before_checkpoint_recreate(neon_env_builder: NeonEnvBuilder):
+# The "pause" case is for a reproducer of issue 6007: an unclean shutdown where we can't do local fs cleanups
+@pytest.mark.parametrize("pause_or_return", ["return", "pause"])
+def test_timeline_init_break_before_checkpoint_recreate(
+    neon_env_builder: NeonEnvBuilder, pause_or_return: str
+):
     env = neon_env_builder.init_start()
     pageserver_http = env.pageserver.http_client()
```

Comment:
```suggestion
):
    ```suggestion
):
    env = neon_env_builder.init_config()
    env.start()
    
    pageserver_http = env.pageserver.http_client()
    pageserver_http.create_tenant(env.initial_tenant)
```